### ISSUE TITLE
M187 Logitech mouse issues

### ISSUE THREAD

**Issue by ghamson** [rsn8887/uae4all2#30]
I got myself an M187 Logitech mouse but it doesn’t seem to be recognized in the Switch version (v1.95).

Is there anything special to turn on in the setup to allow the mouse to work?

A wired USB mouse seems to work without issue.

Hope you can help

**Comment by wronghands** [comment]
Last time I checked it was worded like an extra feature without any promise for compatibility (but the boss can explain better when he's back). Out of the 3 mice I had tested a few weeks ago, only one worked and it was the logitech G700s. Another logitech (similar to yours) and a microsoft one failed but they were more budget-y. So you might be out of luck with that mouse.

**Comment by ghamson** [comment]
It is worded like an extra feature - you are right... was just hoping to have wireless lemmings play.  Do you know what feature of USB is making it compatible?

I have tried a Logitech M215 mouse & the Microsoft Arc Touch mouse (older black version), neither of this work either.... 

My wired HP (Model: MOGIUO) works great however.  I have a few other wireless mouse about... will try those too.

Maybe I can help and get a compatibility list going?  What do you think?

**Comment by rsn8887** [comment]
I wonder if Logitech updated the M187 mouse at some point because mine works. But I bought mine years ago. You did try both USB ports and made sure your M187 worked on the PC right?

**Comment by ghamson** [comment]
Yep, confirmed that it works on a PC and tried all 3 USB ports.  Interestingly my older Logitech M510 wireless mouse is also not working.

However a HP Keyboard and Mouse set with a single receiver works great Mouse Model G33M, Keyboard Model: KBIM - Package product: LV290AA

Amazon Link: https://www.amazon.com/HP-Wireless-Keyboard-LV290AA-ABA/dp/B00501T8XY

I still have a couple of other mice hiding somewhere - so will try them out and report back.

**Comment by rsn8887** [comment]
Interesting. I guess Logitech updated the electronics in the M187 so that newer models don't work anymore.

It would be great to have a compatibility list, since so many mice don't work.

Also very good to hear you got a mouse + keyboard combo to work! I tried all my combos and none worked. But they were all Logitech.

**Comment by ghamson** [comment]
Ok - let me create a spreadsheet on google or something and then perhaps you can link to it in the readme or something... give you rights to up date it also so that we can have multiple contributors.

I will try and get that going today and will post a link here when its available.

Lastly, just want to say thanks for all the work on this... it is by far my favorite homebrew app - been using it since your first version on the Vita... now the Switch... its awesome work.

**Comment by rsn8887** [comment]
Awesome.

I already started a sheet:
https://docs.google.com/spreadsheets/d/1Drbo5-QuSX901MwtOytSMuqRGxeIkq2HELM806I9dj0/edit#gid=0

**Comment by rsn8887** [comment]
And I added it to the release notes.

**Comment by ghamson** [comment]
awesome - I have added a couple more that I have tested since my last message

**Comment by rsn8887** [comment]
Great. I think this will be very useful to people, not just for UAE4all2, but for all homebrew compiled with SDL2.

**Comment by ghamson** [comment]
ok - I think I have exhausted my stash of wireless mice - spreadsheet is updated

**Comment by rsn8887** [comment]
Ok thanks! This helps with Retroarch, too (dosbox core).

**Comment by wronghands** [comment]
Good idea! I've added a few entries. 

Not working:
Logitech | M525 (not a good mouse anyway)
Steelseries | Rival 650 Quantum (this is a legitimate letdown)
Wacom | Intuos Pro Medium (will try as wired)

Working:
Logitech | MX Master 2S (pretty similar to G700s in build but for desktop work instead of gaming)

**Comment by rsn8887** [comment]
My Rii mini x1, mini keyboard and mousepad combo, works fine, too.
